src/LocalRegistry.jl

```diff
@@ -346,10 +346,17 @@ end
 
 function get_remote_repo(package_path, gitconfig)
     git = gitcmd(package_path, gitconfig)
-    remote_name = split(readchomp(`$git remote`), '\n')
-    length(remote_name) > 1 && error("Repository has multiple remotes.")
-    remote_name[1] == "" && error("Repository does not have a remote.")
-    return readchomp(`$git remote get-url $(remote_name[1])`)
+    remote_names = split(readchomp(`$git remote`), '\n')
+    repos = String[]
+    for remote_name in remote_names
+        r = readchomp(`$git remote get-url $(remote_name)`)
+        if !isequal(r,"")
```

Comment:
Although it's possible to define a `git remote` with an empty string as URL, that seems quite useless and I doubt it's worth checking for it. (Note, that's not what the existing comparison with empty string does.)

Reply:
Right. Sorry about that (was rushing). Fixed 👍